src/fparser/two/tests/test_fortran2003.py

```diff
@@ -2110,6 +2110,14 @@ def test_stop_stmt():  # R849
     assert isinstance(obj, tcls), repr(obj)
     assert str(obj) == "STOP 'hey you'"
 
+    obj = tcls('stop "123"//"456"')
```

Comment:
Please could you also test when this extension is disabled.

Reply:
Done.